cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -247,106 +247,56 @@ spec:
     app: {{ template "kubecost.clusterControllerName" . }}
 ---
 # TurndownSchedule Custom Resource Definition for persistence
-{{- if .Capabilities.APIVersions.Has "apiextensions.k8s.io/v1" }}
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: turndownschedules.kubecost.k8s.io
+  name: turndownschedules.kubecost.com
```

Comment:
Are we doing anything to cleanup the old CRD, CRs and permissions?

Reply:
No, does Helm support something like that?

Reply:
No. And what happens to existing ones? Are they migrated by the controller?

Reply:
They are not migrated. We could add that functionality, though I'm a little worried about attempting to touch something namespaced (incorrectly) under `.k8s.io`. Do you have a suggestion?